docker/mint/docker-run.sh

```diff
@@ -6,6 +6,11 @@ set -ex
 export RUST_BACKTRACE=full
 
 declare -a params
+if [ -z "${CHAIN_ID}" ]; then
+  echo "Docker run failed to start, please set required env var CHAIN_ID"
+  exit 1
```

Comment:
Do you need to add this parameter in the command when running the config-builder for faucet? see like 22-24

Reply:
If not needed, I think we don't have to set the env in this script